tests/models/clip/test_text_encoder.py

```diff
@@ -150,12 +141,6 @@ def test_forward_return_hidden_state(self, start):
                 ],
             ]
         )
-        assert_expected(
```

Comment:
shouldn't we have a test that also checks the projected embeddings are as expected?

Reply:
There's another test case `test_forward` checking the projected embeddings are as expected when `return_hidden_state=False`. In this PR, we removed the feature returning both the hidden state and projected embeddings at the same time, hence removing this assert.

Reply:
thanks for the clarification!